frontend/src/lib/components/CreateServiceRequest/UploadFile.svelte

```diff
@@ -9,31 +9,50 @@
 	import { useLibre311Context, useLibre311Service } from '$lib/context/Libre311Context';
 	import type { DropResult } from 'stwui/types';
 	import { page } from '$app/stores';
+	import { stageImage } from '$lib/stores/serviceRequestImageUpload';
+
+	let input: HTMLInputElement;
 
 	export let params: Readonly<Partial<CreateServiceRequestParams>>;
 
 	const dispatch = createEventDispatcher();
 
 	const libre311Service = useLibre311Service();
-	const linkResolver = useLibre311Context().linkResolver;
 
-	let files: FileList;
-
-	$: if (files) {
-		uploadFiles(files);
-	}
+	const linkResolver = useLibre311Context().linkResolver;
 
-	function desktopDropFiles(dropFiles: DropResult) {
-		uploadFiles(dropFiles.accepted);
+	async function onChange() {
+		if (input.files) await uploadImage(input.files[0]);
```

Comment:
We should trigger an error states if input.files.length > 1 .  It basically means that they dragged and dropped multiple files while we only accept one.